src/Silex/Provider/SecurityServiceProvider.php

```diff
@@ -344,14 +371,14 @@ public function register(Container $app)
                 if (is_string($rule[0])) {
                     $rule[0] = new RequestMatcher($rule[0]);
                 } elseif (is_array($rule[0])) {
-                    $rule[0] += [
+                    $rule[0] += array(
```

Comment:
@emirb oh I misunderstand your comment. I thought that you were commenting associates array with string keys. Ok then. :)